tools/utility/dockerise/platform.py

```diff
@@ -119,8 +120,25 @@ def build(repository, platform):
     # If we were building the selected platform then update our selected tag
     if _selected:
         err = subprocess.run(
-            ["docker", "image", "tag", "{}:{}".format(repository, platform), "{}:selected".format(repository)],
+            ["docker", "image", "tag", "{}:{}".format(repository, platform), selected_tag],
             stdout=subprocess.DEVNULL,
         ).returncode
         if err != 0:
             raise RuntimeError("docker image tag returned a non-zero exit code")
+
+
+def pull(repository, platform):
+    # Define our tag strings
+    remote_tag = "{0}/{0}:{1}".format(repository, platform)
+    local_tag = "{0}:{1}".format(repository, platform)
+    selected_tag = "{}:selected".format(repository)
+
+    print("Pulling remote image", remote_tag)
+    err = subprocess.run(["docker", "pull", remote_tag], stderr=DEVNULL, stdout=DEVNULL).returncode
```

Comment:
Might be a good idea to print the output of `docker pull` to console so the user can see that something is actually happening, especially for those with slow internet (like me :stuck_out_tongue:). Possibly with `stdout=sys.stdout`.

Reply:
For sure. Let's print the output (both stdout and stderr) from the subprocess so the user knows what's going on.